system/Makefile.am

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2015 Alon Bar-Lev <alon.barlev@gmail.com>
+#
+MAINTAINERCLEANFILES= $(srcdir)/Makefile.in
+CLEANFILES = \
+	netdata.service \
+	$(NULL)
+
+include $(top_srcdir)/build/subst.inc
+
+SUFFIXES = .in
+
+dist_noinst_DATA = \
```

Comment:
As written the user will get a `netdata.service` for the configuration we package, not one appropriate to their configuration.

I guess this should be made noinst_DATA and then add EXTRA_DIST= netdata-openrc netdata.service.in, or something equivalent.